regression/goto-cc-regression/static-name-clash/test.desc

```diff
@@ -0,0 +1,14 @@
+CORE
+test.c
+--export-function-local-symbols
+^EXIT=0$
+^SIGNAL=0$
+--
+^warning: ignoring
+^Invariant check failed$
+--
+This test guards against a regression added when the local file
+name mangler would not add the required name into the symbol
+table. When the regression was present, we were met with an
+invariant violation during goto-cc compilation. Without it,
+nothing spectacular happens - compilation proceeds as expected.
```

Comment:
Any chance of some more detail or a link to the relevant issue on github?

Reply:
Yes, I have changed it to contain a link to the issue as well.